src/multiselect.js

```diff
@@ -42,12 +42,16 @@ export class Multiselect {
    * to set.
    */
   init(options) {
+    const injectionDiv = this.workspace_.getInjectionDiv();
     this.onKeyDownWrapper_ = Blockly.browserEvents.conditionalBind(
-        this.workspace_.getInjectionDiv(), 'keydown', this, this.onKeyDown_);
+        injectionDiv, 'keydown', this, this.onKeyDown_);
     this.onKeyUpWrapper_ = Blockly.browserEvents.conditionalBind(
-        globalThis['window'], 'blur', this, this.onBlur_);
-    this.onKeyUpWrapper_ = Blockly.browserEvents.conditionalBind(
-        this.workspace_.getInjectionDiv(), 'keyup', this, this.onKeyUp_);
+        injectionDiv, 'keyup', this, this.onKeyUp_);
+    this.onFocusOutWrapper_ = Blockly.browserEvents.conditionalBind(
+        injectionDiv, 'focusout', this, this.onBlur_);
+    injectionDiv.addEventListener('mouseenter', () => {
+      this.workspace_.svgGroup_.parentElement.focus();
```

Comment:
Is there a public accessor for the svgGroup_ field? The field may be obfuscated in a production build.

Reply:
Unfortunately, it seems like they don't have a public accessor for the `svgGroup_` field, and I can see there are also some use in blockly's official plugins:

https://github.com/google/blockly-samples/blob/46dce3f6031c7bb906e992767c17951a91dd1f6d/plugins/scroll-options/src/index.js#L82

Reply:
Issue for this: https://github.com/google/blockly/issues/7157